src-tauri/src/libs/utils.rs

```diff
@@ -28,12 +34,54 @@ impl TimeLogger {
     }
 }
 
-/**
- * Get the app configuration/storage directory
- */
-pub fn get_app_storage_dir() -> PathBuf {
-    let path = dirs::home_dir().expect("Get home dir");
-    path.join(".museeks")
+#[must_use]
+const fn app_name() -> &'static str {
+    // Must match the value of "identifier" in tauri.config.json!
+    "io.museeks.app"
+}
```

Comment:
you should get that from Tauri: https://docs.rs/tauri/2.0.0-beta.11/tauri/struct.Config.html#structfield.identifier